File: src/main/java/com/extendedae_plus/mixin/impl/bridge/HelperProviderUpgradesInv.java
package com.extendedae_plus.mixin.impl.bridge;

import appeng.api.upgrades.IUpgradeInventory;

public interface HelperProviderUpgradesInv {
    IUpgradeInventory eaep$getUpgradeInventory();

    void eaep$addAction(Runnable action);
}

File: src/main/kotlin/com/extendedae_plus/mixin/helper/HelperProviderUpgradesInv.kt
package com.extendedae_plus.mixin.helper

import appeng.api.upgrades.IUpgradeInventory

interface HelperProviderUpgradesInv {
    fun `eaep$getUpgradeInventory`(): IUpgradeInventory

    fun `eaep$addAction`(action: Runnable)
}
